src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/Microsoft.Extensions.Configuration.Binder.targets

```diff
@@ -0,0 +1,28 @@
+<Project InitialTargets="NETStandardCompatError_Microsoft_Extensions_Configuration_Binder">
+    <Target Name="_Microsoft_Extensions_Configuration_BinderRemoveAnalyzer" 
+            Condition="'$(EnableConfigurationBindingGenerator)' != 'true'"
+            AfterTargets="ResolvePackageDependenciesForBuild;ResolveNuGetPackageAssets">
+
+        <ItemGroup>
+            <Analyzer Remove="@(Analyzer->WithMetadataValue('NuGetPackageId', 'Microsoft.Extensions.Configuration.Binder'))" />
+        </ItemGroup>
+    </Target>
+
+    <!-- Only apply for [net4.6.1] and [netcoreapp2.0, net6.0) -->
+    <Target Name="NETStandardCompatError_Microsoft_Extensions_Configuration_Binder"
+          Condition="'$(SuppressTfmSupportBuildWarnings)' == '' AND
+          (
+            (
+                $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net4.6.1')) AND
+                !$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net4.6.2'))
+            )
+            OR
+            (
+                $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netcoreapp2.0')) AND
+                !$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))
+            )
+          )
+          ">
+    <Warning Text="Microsoft.Extensions.Configuration.Binder doesn't support $(TargetFramework) and has not been tested with it. Consider upgrading your TargetFramework to net6.0 or later. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk." />
```

Comment:
Looks like we copied the `net6.0` warning text. We should fix this up so that it lists the right thing.  If we want to be specific we could pull the conditions into the target.

Reply:
Here's a suggestion,
```xml
    <Target Name="NETStandardCompatError_Microsoft_Extensions_Configuration_Binder"
          Condition="'$(SuppressTfmSupportBuildWarnings)' == ''">
        <PropertyGroup>
            <_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework 
                Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netcoreapp2.0')) AND 
                           !$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))"
                >net6.0</_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework>
            <_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework 
                Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net461')) AND 
                           !$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net462'))"
                >net462</_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework>
        </PropertyGroup>
        <Warning Condition="'$(_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework) != ''"
                Text="Microsoft.Extensions.Configuration.Binder doesn't support $(TargetFramework) and has not been tested with it. Consider upgrading your TargetFramework to $(_Microsoft_Extensions_Configuration_Binder_Compatible_TargetFramework) or later. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk." />
    </Target>
 ```